packages/openapi/src/rpcRequest.ts

```diff
@@ -102,11 +102,15 @@ class ClientRPCRequest implements RPCRequest {
 }
 
 function isServerEvent(event: Event): event is ServerRpcEvent {
-  return !!event.httpServerRequest && !!event.httpServerResponse;
+  return (
+    !!event.httpServerRequest &&
+    !!event.httpServerResponse &&
+    !!(event.httpServerRequest.normalized_path_info || event.httpServerRequest?.path_info)
```

Comment:
This is actually the fix, here. The AppMap contains a request with `path_info: ''` and no `normalized_path_info`. `''` is falsey in JS.